node/cn/protocol.go

```diff
@@ -117,23 +115,6 @@ var errorToString = map[int]string{
 	ErrFailedToGetStateDB:      "Failed to get stateDB",
 }
 
-//go:generate mockgen -destination=node/cn/mocks/txpool_mock.go -package=mocks github.com/klaytn/klaytn/node/cn TxPool
-// TxPool is an interface of blockchain.TxPool used by ProtocolManager.
-type TxPool interface {
-	// HandleTxMsg should add the given transactions to the pool.
-	HandleTxMsg(types.Transactions)
-
-	// Pending should return pending transactions.
-	// The slice should be modifiable by the caller.
-	Pending() (map[common.Address]types.Transactions, error)
-
-	CachedPendingTxsByCount(count int) types.Transactions
-
-	// SubscribeNewTxsEvent should return an event subscription of
-	// NewTxsEvent and send events to the given channel.
-	SubscribeNewTxsEvent(chan<- blockchain.NewTxsEvent) event.Subscription
-}
-
```

Comment:
It seems that this `TxPool` interface is integrated with `work.TxPool`. 
Why do you merge them? Merged interface is better than separated interface for several packages?

Reply:
@KimKyungup Thanks for the comment. As `cn.CN` and `sc.ServiceChain` implement `work.Backend` interface, they need to use the same type for `work.TxPool`. This is the reason why `work.TxPool` interface has been introduced.

Reply:
I misunderstood. :) Thanks.